java/trace/src/main/scala/org/typelevel/otel4s/java/trace/SpanBackendImpl.scala

```diff
@@ -89,7 +90,7 @@ private[java] class SpanBackendImpl[F[_]: Sync](
     }
 
   private[otel4s] def end: F[Unit] =
-    Sync[F].delay(jSpan.end())
+    Sync[F].realTime.flatMap(now => end(now))
```

Comment:
Only `startUnmanaged` is affected. For resource-like API, we were injecting the realtime under the hood. E.g. https://github.com/typelevel/otel4s/pull/122/files#diff-988bfe14df2f23c008240b2752bde4a711b0c4fa716b62f5416f1de82582bc55L330